build.sc

```diff
@@ -350,7 +350,8 @@ trait CliIntegrationBase extends SbtModule with ScalaCliPublishModule with HasTe
            |  def scala213 = "${Scala.scala213}"
            |  def scala3   = "${Scala.scala3}"
            |  def defaultScala = "${Scala.defaultUser}"
-           |
+           |  def bloopVersion = "${Deps.bloopConfig.dep.version}"
+           |  def oldBloopVersion = "${Deps.oldBloopConfig.dep.version}"
```

Comment:
Do we need to write the old bloop version here? I think it's fine to hardcode it in the tests…

Reply:
Done